src/extensibility/ExtensionManager.js

```diff
@@ -260,6 +280,18 @@ define(function (require, exports, module) {
             if (!extensions[id]) {
                 extensions[id] = {};
             }
+            
+            if (locationType === LOCATION_USER) {
+                if (!userExtensions[id]) {
+                    userExtensions[id] = {};
+                }
+                
+                userExtensions[id].installInfo = {
```

Comment:
@MarcelGerber Currently also we have other properties like registryInfo and most probably others can be added.